yellow-paper/docs/public-vm/avm-circuit.md

```diff
@@ -3,83 +3,137 @@ sidebar_position: 1
 ---
 
 # AVM Circuit
+The AVM circuit's purpose is to prove execution of the correct sequence of instructions for a public execution request. Regardless of whether execution succeeds or reverts, the circuit always generates a valid proof of execution.
```

Comment:
I would remove the word "correct" as it is not so clear to me what correct means. As said later, the VM should provide a valid proof even if the sequence of instructions lead to an error (division by zero, etc,...)